config/config.ts

```diff
@@ -0,0 +1,15 @@
+import * as fs from 'fs'
+
+let configFile
+let localConfig
+
+// If project is being run locally, the developer will have their API key in the json file
+try {
+  configFile = fs.readFileSync('./config/hollowbot.config.json', 'utf8')
+  localConfig = JSON.parse(configFile)
```

Comment:
This could throw due to malformed JSON object. The error message below would be misleading if that happened.